Configuration/AlCa/python/autoCondModifiers.py

```diff
@@ -11,8 +11,23 @@ def autoCond0T(autoCond):
     ConditionsFor0T =  ','.join( ['RunInfo_0T_v1_mc', "RunInfoRcd", connectionString, "", "2020-07-01 12:00:00.000"] )
     GlobalTags0T = {}
     for key,val in six.iteritems(autoCond):
-        if "phase" in key:    # restric to phase1 upgrade GTs
+        if "phase" in key:    # restrict to phase1 upgrade GTs
             GlobalTags0T[key+"_0T"] = (autoCond[key], ConditionsFor0T)
 
     autoCond.update(GlobalTags0T)
     return autoCond
+
+def autoCondHLTHI(autoCond):
+
+    GlobalTagsHLTHI = {}
+
+    # emulate hybrid ZeroSuppression on the VirginRaw data of 2015
+    FullPedestalsForHLTHI =  ','.join( ['SiStripFullPedestals_GR10_v1_hlt', "SiStripPedestalsRcd", connectionString, "", "2021-03-11 12:00:00.000"] )
+    MenuForHLTHI =  ','.join( ['L1Menu_CollisionsHeavyIons2015_v5_uGT_xml', "L1TUtmTriggerMenuRcd", connectionString, "", "2021-03-11 12:00:00.000"] )
+
+    for key,val in six.iteritems(autoCond):
+        if key == 'run2_hlt_relval':    # modification of HLT relval GT
+            GlobalTagsHLTHI['run2_hlt_hi'] = (autoCond[key], FullPedestalsForHLTHI, MenuForHLTHI)
```

Comment:
@icali @CesarBernardes please take note of this. This GlobalTag, suitable for Hybrid ZeroSuppression emulation on 2015 Heavy Ion ViriginRaw data has become symbolic. This means it's fine for local tests and RelVals but not e.g. for central samples production. Please comment if this might constitue a problem.

Reply:
Of course, if it is desired to reconstruct 2015 HI data in 11_2_X or later, AlCa could trivially create a regular, non-symbolic GT. Our assumption is that this would be needed rarely enough that it is simpler to maintain the GT as a symbolic GT based on the `run2_hlt_relval` GT.